app/views/hotspots/_show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h4>Donation Sights</h4>
+<div class="row">
+    <div class="col-xs-0"></div>
+<div id = 'show_form' class="col-xs-5">
```

Comment:
Indent this whole div to be even with line 3, since this is inside the "row" div defined on line 2